UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -122,4 +128,10 @@ public void CancelJump()
     {
         isJumping = false; //This will stop the reduction to the gravity, which will then quickly pull down the character
     }
-}
\ No newline at end of file
+
+    private void OnControllerColliderHit(ControllerColliderHit hit)
+    {
+        hitNormal = hit.normal;
+        canJump = (Vector3.Angle(Vector3.up, hitNormal) <= slopeLimit);
```

Comment:
You could use characterController.slopeLimit here instead of an variable. So you still could use the editor to try out different values during runtime.

If you don't like that then adding a SerializedField-Attribut to the variable may would be a compromise.